### DISCUSSION TITLE
Native modules willl be deprecated?

### DISCUSSION THREAD

**Discussion by Akashmic** [reactwg/react-native-new-architecture#169] [Q&A] [answered]
User native module will be deprecated in future whne introduction of bridgeless mode

**Comment by cortinico** [comment, marked as answer]
Nope, we're not looking into deprecating the old APIs in the immediate future.
We developed interop layers that let you use old Native Modules/Components in apps running on the New Architecture.

Obviously the future of React Native is the New Architecture, so we will eventually deprecate and remove those older APIs but that's something we haven't planned yet.
